style.css

```diff
@@ -331,3 +347,115 @@ input:focus {
   color: #fff;
   transition: all 0.8s;
 }
+
+@media (min-width: 768px) {
```

Comment:
- [ ] :red_circle:  The mobile version of your portfolio seems to be affected by the modifications you have made. Kindly try to fix these issues. 
  -  The content does not fit and you have to use the horizontal scroll bar to see everything
  -  Please make sure you set a fixed width on each image in your cards.
  -  Please make sure on the mobile screen, only the hamburger menu appears and not the menu items


Your Mobile version

![affectected](https://user-images.githubusercontent.com/26736582/199824161-5c288d37-a2f5-41bd-a522-9d4f5fe127d5.gif)